src/goto-instrument/contracts/instrument_spec_assigns.cpp

```diff
@@ -27,8 +27,62 @@ Date: January 2022
 
 #include "utils.h"
 
-///// PUBLIC METHODS /////
+void add_pragma_disable_pointer_checks(source_locationt &location)
+{
+  location.add_pragma("disable:pointer-check");
+  location.add_pragma("disable:pointer-primitive-check");
+  location.add_pragma("disable:pointer-overflow-check");
+  location.add_pragma("disable:signed-overflow-check");
+  location.add_pragma("disable:unsigned-overflow-check");
+  location.add_pragma("disable:conversion-check");
+}
```

Comment:
What is the motivation to move this away from `utils` and add it in here? Perhaps it's only applied to assigns clauses today, but there might be value in using it in other places in the future.

Reply:
Same for other `pragma` instrumentation related changes.

Reply:
all these pragmas are solely used in assigns clause checking, no need to expose them in utils.